website/client-old/js/services/questServices.js

```diff
@@ -104,6 +104,9 @@ angular.module('habitrpg')
             Groups.data.party = party;
             $state.go('options.social.party');
             resolve();
+            if ($state.current.name === "options.social.party") {
+              $state.reload();
```

Comment:
I'm ambivalent about it. What you have already is fine with me as a stop gap until the front end redesign is complete.